src/mixins/boundary.js

```diff
@@ -0,0 +1,60 @@
+import * as THREE from "three";
+
+export const BoundaryMixin = (superclass) => class extends superclass {
+
+    constructor(config) {
+        super(config);
+        this.boundaryConditions = config.boundaryConditions || {};
```

Comment:
We should explain the schema for boundaryConditions

Reply:
Done.